setup.py

```diff
@@ -60,11 +61,17 @@ def read(*rnames):
     keywords='restricted execution security untrusted code',
     author='Zope Foundation and Contributors',
     author_email='zope-dev@zope.org',
+    project_urls={
+        "Documentation": "https://restrictedpython.readthedocs.io/",
+        "Source": "https://github.com/zopefoundation/RestrictedPython",
+        "Tracker": "https://github.com/zopefoundation/RestrictedPython/issues",
+    },
     packages=find_packages('src'),
     package_dir={'': 'src'},
     install_requires=[
         'setuptools',
     ],
+    python_requires=">=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*, !=3.4.*, <4 ",
```

Comment:
```suggestion
    python_requires=">=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*, !=3.4.*, <3.9 ",
```

This way we make sure it does not run with an untested Python version.

Reply:
This would mean for us, that we need to adjust or work on special branches for each new Python Version, but ok, I will change